Week2/1.Hello World.js

```diff
@@ -0,0 +1,5 @@
+// 1
+let language1 = 'Merhaba dunya!' ,
```

Comment:
Can you also add which languages these are?

And do you think it's better to use `let` or `const`?